linear_actuator_type: firgelli

linear_actuator_name: &linear_actuator_joint linear_actuator_joint

joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

linear_actuator_controller:
  type: position_controllers/JointPositionController
  joint: *linear_actuator_joint
